src/Our.Umbraco.RobotsTxtEditor/Web/UI/App_Plugins/RobotsTxtEditor/views/robotstxteditor.html

```diff
@@ -0,0 +1,40 @@
+<div class="umb-robotstxteditor" ng-controller="RobotsTxtEditorController as vm">
+
+    <umb-box>
+        <umb-box-content>
+            
+            <h3 class="bold">Robots.txt Editor</h3>
+
+            <umb-load-indicator ng-show="vm.loading"></umb-load-indicator>
+
+            <div class="umb-robotstxteditor-main" ng-show="!vm.loading">
+                <div class="umb-robotstxteditor-controls">
+                    <p>
+                        <!-- TODO: replace with 'posh' textarea with line numbers (like editing styles)-->
+                        <textarea ng-model="vm.data.FileContents"></textarea>
+                    </p>
+                    <p>
+                        <umb-button type="button"
+                                    label="{{vm.data.FileExists ? 'Update Robots.txt' : 'Create Robots.txt'}}"
+                                    button-style="success"
+                                    action="vm.saveData()">
+                        </umb-button>
+
+                    </p>
+                </div>
+                <div class="umb-robotstxteditor-notes">
+                    <!-- TODO: add button to insert 'standard' rules for an Umbraco website -->
+                    <!-- TODO: add other buttons / helpful functionality -->

```

Comment:
📓 Other features from the old version were...

1. Insert User-Agent rule
2. Insert Disallow rule
3. Comment out rules
4. Uncomment rules

**JS**: https://github.com/leekelleher/umbraco-robotstxt-editor/blob/master/Our.Umbraco.Tree.RobotsTxt/umbraco/plugins/robots-txt/editRobotsTxtFile.aspx#L5-L32
**C#**: https://github.com/leekelleher/umbraco-robotstxt-editor/blob/master/Our.Umbraco.Tree.RobotsTxt/umbraco/plugins/robots-txt/editRobotsTxtFile.aspx.cs#L87-L109